packages/tutorials/election-dapp/front-end/package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "@tutorials/election",
```

Comment:
Is the `@tutorials` vendor/namespace something we own/want in the repo?

Reply:
Maybe not, I'll change it to `@kadena/...`